### DISCUSSION TITLE
`OffsetDateTime::now_local()` Cannot get the correct local time

### DISCUSSION THREAD

**Discussion by ayangweb** [time-rs/time#713] [Question]
https://github.com/tauri-apps/plugins-workspace/issues/1926

I hope I can get some help.

**Comment by jhpratt** [comment]
The method call is not guaranteed to succeed. In this situation, it returns an error, almost certainly because your program is multi-threaded. The fallback to UTC is not the behavior of `time`, but of `tauri`.

**Comment by ayangweb** [comment]
thank you
